apps/wallet/src/ui/app/pages/layout/index.tsx

```diff
@@ -38,6 +38,8 @@ function PageLayout({
                 )}
             >
                 {children}
+                <div id="overlay-portal-container"></div>
```

Comment:
you should only need one dom node here for portals to render into... is that not the case?

Reply:
oh I mean we could technically have one portal container, I figured I'd make specific portal containers so things would be generally more isolated and moveable in the future. If you have a strong preference, I can just have one portal container called `main-page-layout-portal-container`